### PR TITLE
✏ Reword confusing sentence in docs file `typo-fix-path-params-numeric-validations.md`

### PR DESCRIPTION
**Pull request by ccrenfroe**
A simple update to what was previously an improper and confusing sentence. 

Thanks to @Kludex  for the new sentence recommendation!

### CODE REVIEW

File: docs/en/docs/tutorial/path-params-numeric-validations.md [hunk 1]
```diff
@@ -106,9 +106,9 @@ And you can also declare numeric validations:
 * `le`: `l`ess than or `e`qual
 
 !!! info
-    `Query`, `Path`, and others you will see later are subclasses of a common `Param` class (that you don't need to use).
+    `Query`, `Path`, and others classes you will see later are subclasses of a common `Param` class.
```
Comment (paxcodes) [inline docs/en/docs/tutorial/path-params-numeric-validations.md:109]: "...other classes..."

File: docs/en/docs/tutorial/path-params-numeric-validations.md [hunk 2]
```diff
@@ -106,9 +106,9 @@ And you can also declare numeric validations:
...
 
-    And all of them share the same all these same parameters of additional validation and metadata you have seen.
+    All of them share the same parameters for additional validation and metadata.
```
Comment (paxcodes) [inline docs/en/docs/tutorial/path-params-numeric-validations.md:111]: IIUC, everything under "Tutorial - User Guide" is intended to be read chronologically from top-to-bottom. In light of this intention, "you have seen" makes sense and I wouldn't remove it. I would add "so far in the tutorial" to make this intention explicit.

Comment (ccrenfroe) [reply docs/en/docs/tutorial/path-params-numeric-validations.md:111]: Hey @paxcodes 👋 

I think these are great tweaks and I'll make sure to get them in. Thanks for the clarity.

**Comment by ccrenfroe** [comment]
Been a while, how is it now? @paxcodes I believe I addressed your suggestion.

**Comment by paxcodes** [comment]
LGTM but I don't have any rights to this repo. I'm a fellow FastAPI user.

**Comment by tiangolo** [comment]
Awesome, thanks @ccrenfroe! :cake: 

And thanks for the help @paxcodes! :rocket:

**Pull request merged**